src/Microsoft.Extensions.Configuration.UserSecrets/ConfigurationExtensions.cs

```diff
@@ -40,7 +39,7 @@ public static IConfigurationBuilder AddUserSecrets<T>(this IConfigurationBuilder
         /// </summary>
         /// <param name="configuration"></param>
         /// <returns></returns>
-        [Obsolete("This method is obsolete and will be removed in a future version. The recommended alternative is Microsoft.Extensions.Configuration.ConfigurationExtensions.AddUserSecrets<T>() or Microsoft.Extensions.Configuration.ConfigurationExtensions.AddUserSecrets(string userSecretsId).")]
+        [Obsolete("This method is obsolete and will be removed in a future version. The recommended alternative is Microsoft.Extensions.Configuration.ConfigurationExtensions.AddUserSecrets(this IConfigurationBuilder configuration, string userSecretsId).")]
```

Comment:
Also, I understand I missed a conversation about this, but how are customers going to figure out how to use this overload? Do they need to go extract the user secrets Id from its present location? I am worried that people may need to read the manual for this then the whole migration from project.json to csproj will be harder.